src/lexer/mod.rs

```diff
@@ -106,9 +110,9 @@ impl Lexer {
       if !self.tokenise_json() {
         value = Some(self.src[position + 1..self.position - 1].into());
       }
-    } else if self.current() == '/' && self.is_newline {
+    } else if self.current() == '`' && self.is_newline {
```

Comment:
We don't need to check for a newline here any more, as the backticks are used as delimiters instead